dietpi/dietpi-software

```diff
@@ -6547,6 +6560,21 @@ Mycroft AI requires at least 2 GiB memory on first start. We will now increase y
 
 		fi
 
+		software_id=177 # TasmoAdmin
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+			DEPS_LIST="$PHP_NAME-curl $PHP_NAME-zip" # https://github.com/reloxx13/TasmoAdmin#linux
+
+			Download_Install 'https://github.com/reloxx13/TasmoAdmin/archive/master.zip'
+			cp -a TasmoAdmin-master/tasmoadmin/ /var/www/tasmoadmin/
+			rm -R TasmoAdmin-master
+			chown -R www-data:www-data /var/www/tasmoadmin
+
+			[[ $DEPS_LIST ]] && G_DIETPI-NOTIFY 2 'Installing required PHP modules' && G_AGI $DEPS_LIST
```

Comment:
`DEPS_LIST` packages, when assigned, are installed automatically by the `Download_Install` function, parallel to the archive download, so no need to do this manually.

Reply:
So if I understand correctly I can change it to:
`G_DIETPI-NOTIFY 2 'Installing required PHP modules'`

Reply:
I've removed line 6574.
What does G_AGI do? Do I need to let it there?

Reply:
@svh1985 
G_AGI == non-interactive apt-get install, so when Download_Install is skipped, it should be called instead.
`[[ $DEPS_LIST ]] && G_DIETPI-NOTIFY 2 'Installing required PHP modules' && G_AGI $DEPS_LIST` can then be removed.

Reply:
Okay clear, I've removed it!